Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -29,6 +30,7 @@ class Decoder : public VideoDecoder {
     ~Decoder() override { }
     /* (8.1) General */
     DecoderErrorOr<void> receive_sample(ReadonlyBytes) override;
+    static bool is_codec_supported(CodecID codec) { return codec == CodecID::VP9; }
```

Comment:
This is unused currently, so we should omit it for now. We may have a use for it in the future, but we can just add it then.